README.md

```diff
@@ -44,39 +44,47 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
   
 ## Index
 
-- [Android Development](#android-development)
-- [Backend](#backend)
-  * [Python](#python)
-  * [Django](#django)
-  * [Go](#go)
-  * [NodeJs](#nodejs)
-  * [Express](#express)
-  * [NextJs](nextjs)
-  * [Java](#java)
-  * [Spring Boot](#spring-boot)
-  * [Shell Commands](#shell-commands)
-  * [Laravel](#laravel)
-- [Blog writing](#blog-writing)
-- [Core-CS](#core-cs)
-- [Design](#design)
-- [Docker](#docker)
-- [Frontend](#frontend)
-  * [HTML](#html)
-  * [CSS](#css)
-  * [CSS Frameworks](#css-frameworks)
-  * [JavaScript](#javascript)
-  * [API](#api)
-  * [React](#react)
-  * [Angular](#angular)
-  * [Ember](#ember)
-  * [Practice](#practice)
-- [Git and GitHub](#git-and-github)
-- [Good first issue finder](#good-first-issue-finder)
-- [Linux](#linux)
-- [Machine Learning](#machine-learning)
-- [Vue](#vue)
-- [Web3](#web3)
-  * [Blockchain](#blockchain)
+- [:wave:  Code For Community welcomes you](#wave--code-for-community-welcomes-you)
+- [Gateway to Open Source | Best Programming Resources](#gateway-to-open-source--best-programming-resources)
+  - [Agenda:rocket:](#agendarocket)
+  - [Share your contribution](#share-your-contribution)
+    - [**Join Community :people_holding_hands: & Connect with me :handshake:**](#join-community-people_holding_hands--connect-with-me-handshake)
+  - [Contribution Guide :point_down:](#contribution-guide-point_down)
```

Comment:
remove lines 47 to 52, Please review your PR at your end before making it

![Screenshot from 2022-10-08 21-59-02](https://user-images.githubusercontent.com/75534912/194717697-3520c588-0bfb-431a-b0e2-e5ab1644e419.png)